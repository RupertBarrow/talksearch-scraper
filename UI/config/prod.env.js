'use strict'
module.exports = {
  NODE_ENV: '"production"',
  API_URL: '"https://algolia-talksearch.herokuapp.com"'
}
