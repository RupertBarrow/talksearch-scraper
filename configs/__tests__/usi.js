import config from '../usi.js';
import helper from '../config-helper.js';

describe('USI', () => {
  describe('transformData', () => {
    let current;
    beforeEach(() => {
      current = input => config.transformData(input, helper);
    });

    it("Pour faire naître une idée - Cédric Villani, à l'USI", () => {
      const input = {
        video: {
          title: "Pour faire naître une idée - Cédric Villani, à l'USI",
        },
      };

      const actual = current(input);

      expect(actual).toMatchSnapshot();
    });
  });
});
