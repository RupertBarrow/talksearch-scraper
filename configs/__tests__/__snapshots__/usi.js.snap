// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`USI transformData Pour faire naître une idée - Cédric Villani, à l'USI 1`] = `
Object {
  "speakers": Array [
    Object {
      "name": "Cédric Villani",
    },
  ],
  "video": Object {
    "title": "Pour faire naître une idée",
  },
}
`;
